config/iam-policy-templates/upload-api-lambda.json

```diff
@@ -79,7 +68,8 @@
       ],
       "Resource": [
         "arn:aws:sqs:us-east-1:$account_id:dcp-upload-pre-csum-queue-${DEPLOYMENT_STAGE}",
-        "arn:aws:sqs:us-east-1:$account_id:dcp-upload-area-deletion-queue-${DEPLOYMENT_STAGE}"
+        "arn:aws:sqs:us-east-1:$account_id:dcp-upload-area-deletion-queue-${DEPLOYMENT_STAGE}",
+        "arn:aws:sqs:us-east-1:$account_id:dcp-upload-pre-batch-validation-queue-${DEPLOYMENT_STAGE}"
```

Comment:
Why "pre" batch validation queue?  I guess this indicates that this is the queue before the validation process, and there may one day be a "post" batch validation queue, but I think names like "batch-validation-queue" and "batch-validation-results" would work better.

Reply:
Actually all validation is done in Batch, so nix that too.
Name it "dcp-upload-validation-queue"

Reply:
Sounds good